R/tbl_regression.R

```diff
@@ -333,17 +333,18 @@ estimate_header <- function(x, exponentiate) {
   }
 
   # assigning header and footer ------------------------------------------------
+  language <- get_theme_element("pkgwide-str:language", default = "en")
   if (model_type == "logistic") {
     header <- ifelse(exponentiate == TRUE, "OR", "log(OR)")
-    attr(header, "footnote") <- "OR = Odds Ratio"
```

Comment:
@oranwutang @karissawhiting what do you think about allowing the abbreviations to be updated as well?  I think that should be ok, right?

Reply:
At least in spanish it won't be necessary, as a matter of facts, in Spanish OR is usuallly abbreviated "OR" and although the translation is "razón de probabilidades" many people use "odds ratio" as well (same with hazard ratio)